src/controlCenter/controlCenter.vala

```diff
@@ -113,6 +113,11 @@ namespace SwayNotificatonCenter {
                 return false;
             });
             this.box.add (new TopAction ("Do Not Disturb", dnd_button, false));
+
+            if (ConfigModel.instance.notification_center_height != 0) {
+                this.default_height = ConfigModel.instance.notification_center_height;
+            }
+            this.default_width = ConfigModel.instance.notification_center_width;
```

Comment:
Needs to be set in the set_anchor method to be able to update on a config refresh